Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2292,6 +2297,8 @@ extension BrowserViewController: WKNavigationDelegate {
                     UIApplication.shared.openURL(url)
                 }))
                 present(alert, animated: true, completion: nil)
+
+                LeanplumIntegration.sharedInstance.track(eventName: .openedTelephoneLink)
```

Comment:
This always fires when the user taps a `tel:`, `facetime:` or `facetime-audio:` link. The event name says *Telephone*. So is that correct?

Also, I wonder if this event should only fire if the user actually hits *OK* in the dialog that we show to confirm opening the link.